packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -406,6 +406,12 @@ export default class Editor implements IEditor {
     }
 
     public select(arg1: any, arg2?: any, arg3?: any, arg4?: any): boolean {
+        if ((<HTMLTableElement>arg1)?.rows != undefined) {
+            const selection = this.core.api.selectTable(this.core, arg1, arg2);
+            this.core.domEvent.tableSelectionRange = selection;
+            return;
+        }

```

Comment:
question: how can I unselect a table?

We should always unselect existing table if any when we call select. Say currently  a table is selected, then I want to select something else, then we can call selectTable() with null parameter to unselect first, then call selectRange(), same for the other way

Reply:
I see, adding this if the first param in select api is not table, unselect the table selection.